tbc_error_page/broken.json

```diff
@@ -0,0 +1,130 @@
+(lp1
```

Comment:
Do these json files need to be checked in to the repo? if not, can these be specifically added to gitignore?